libcxx/docs/ReleaseNotes/18.rst

```diff
@@ -35,9 +35,25 @@ see the `releases page <https://llvm.org/releases/>`_.
 What's New in Libc++ 18.0.0?
```

Comment:
Can you please also include the additions to the release notes from https://github.com/llvm/llvm-project/pull/78561 ?

Reply:
Do you mean adding the committed changes or separate changes?